contracts/teller-network/MetadataStore.sol

```diff
@@ -99,7 +99,7 @@ contract MetadataStore is MessageSigned {
         uint _nonce,
         bytes memory _signature
     ) internal view returns(address) {
-        return recoverAddress(getSignHash(_dataHash(_username, _statusContactCode, _nonce)), _signature);
+        return _recoverAddress(_getSignHash(_dataHash(_username, _statusContactCode, _nonce)), _signature);
```

Comment:
Can this be a couple more lines, but with temporary variables so that it's easier to read?